packages/kolibri-common/components/SearchFiltersPanel/AccordionSelectGroup.vue

```diff
@@ -330,9 +338,21 @@
         // Takes the dot separated category value and checks if it is active
         return this.activeCategories.some(k => k.includes(categoryValue));
       },
-      categoryIcon() {
-        // TODO Add icons to KDS then use them
-        return 'categories';
+      categoryIcon(category) {
+        if (category === 'WORK') {
+          return 'skillsResource';
+        } else if (category === 'FOUNDATIONS') {
+          return 'basicSkillsResource';
+        }
+        // for those with a clearer 1:1 match with the category and icon
+        else {
+          const categoryWordsString = category.toLowerCase().split('_');
+          for (let i = 1; i < categoryWordsString.length; i++) {
```

Comment:
With this we are transforming snake case to camel case here, right? I see that above in this file we use the lodash/camelCase function to [transform some other snake case strings](https://github.com/marcellamaki/kolibri/blob/7fc78d4e0fb82c09ad3fb62f79665bfa8d19c8db/packages/kolibri-common/components/SearchFiltersPanel/AccordionSelectGroup.vue#L286), I think we can also use this here.

Reply:
Updated, @AlexVelezLl - thank you for the tip here!